docker/initial-data.json

```diff
@@ -371,20 +372,25 @@
         "id": 5,
         "name": "Delegates",
         "permissions": [
-            "agenda_item.can_see",
-            "agenda_item.can_see_internals",
-            "agenda_item.can_be_speaker",
+            "agenda_item.can_see_internal",
+            "list_of_speakers.can_be_speaker",
             "assignment.can_nominate_other",
             "assignment.can_nominate_self",
-            "assignment.can_see",
             "meeting.can_see_frontpage",
             "projector.can_see",
             "mediafile.can_see",
-            "motion.can_create",
             "motion.can_manage",
```

Comment:
this one is wrong - a delegate does not have manage perms. It should be these perms: `motion.can_create`, `motion.can_create_amendments`, `motion.can_support`

Reply:
This resolves the discussion below

Reply:
done